hack/.ci/prepare_release

```diff
@@ -78,6 +78,7 @@ cp -R "$repo_root_dir" "$REPO_PATH"
 
 current_dir="$(pwd)"
 cd "$REPO_PATH"
+# TODO: can we drop this? Or do we rely on this to install stuff?
 make install-requirements
```

Comment:
Thanks for the pointer!
I pushed another commit which executes `make install-requirements` if the target is present and skips it otherwise to make the migration more convenient and flexible.